build.target.xml

```diff
@@ -197,6 +197,7 @@ if="deploy.tomcat.required">
 			<pathelement location="${dist.lib.ext.dir}/yuicompressor.jar"/>
 			<pathelement location="${dist.lib.ext.dir}/commons-io.jar"/>
 			<pathelement location="${dist.lib.ext.dir}/closure-compiler.jar"/>
+			<pathelement location="${dist.lib.ext.dir}/closure-compiler-unshaded.jar"/>
```

Comment:
Option 1: Keep closure-compiler.jar JUST for zipjs, should be updated separately (might be forgotten)

Option 2: Use closure-compiler-unshaded.jar and add its dependencies (PERFERRED)